pkg/install/install.go

```diff
@@ -109,6 +109,11 @@ func RunStatus(cmd *cobra.Command, args []string) {
 	log.Printf("Operator Status:")
 	operator.RunStatus(cmd, args)
 	log.Printf("")
+	log.Printf("System Wait Ready:")
+	if system.WaitReady() {
```

Comment:
just nitpicking here, but what happens when there is no noobaa CR or CRD installed?

Reply:
It Woll get into an endless loop of:
```
⏳ Failed to get system: no matches for kind "NooBaa" in version "noobaa.io/v1alpha1"
```
But that is no different from a situation where it hangs on connecting or configuring, which can happen on install also. 
We can add to the `WaitReady` timeout if we want.